Chart.yaml

```diff
@@ -15,15 +15,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.2.3
```

Comment:
seems to not be consistent with the changelog. Could all changes in this MR be put together into 0.2.1?

Reply:
Need to look into this there might be a change that I forgot to add into the Changelog